### PR TITLE
In Class Assignment - Chris Lagunilla

### PR DESCRIPTION
**Pull request by chrislagunilla**
My HTML/CSS for the in class assignment

### CODE REVIEW

File: css/icOne.css [hunk 1]
```diff
@@ -1,3 +1,82 @@
 body, html {
   font-size: 12px;
-}
\ No newline at end of file
+}
+
+p {
+  margin-left: 50px;
```
Comment (mattbowytz) [inline css/icOne.css]: shorthand is better - `margin: 0 50px;` would translate to the same thing

File: css/icOne.css [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+  margin-right: 50px;
+}
+
+nav {
+  text-align: center;
+  border-style: double;
+}
+
+nav ul li {
+  font-family: Helvetica;
```
Comment (mattbowytz) [inline css/icOne.css]: you may want to consider fallbacks in the future, Helvetica is almost universally accepted though

File: css/icOne.css [hunk 3]
```diff
@@ -1,3 +1,82 @@
...
+  font-size: 2rem;
+  list-style: none;
+  display: inline;
+  margin-left: 10px;
+  margin-right: 10px;
```
Comment (mattbowytz) [inline css/icOne.css]: shorthand

File: index.html [hunk 4]
```diff
@@ -2,10 +2,60 @@
...
+    </section>
+
+    <section id="sectionThree">
+      <p class="green">Etiam vulputate est quis sapien eleifend suscipit. Cras sagittis dignissim sem sit amet volutpat. Suspendisse blandit rutrum risus vel hendrerit. Aliquam erat volutpat. Curabitur molestie et lectus in posuere. Suspendisse imperdiet sodales lacus at accumsan. Praesent fringilla ornare eros, in sodales augue maximus iaculis. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Cras finibus eros mauris, id ornare sapien consectetur eget. Proin sollicitudin consectetur risus. Pellentesque felis mi, sodales non mauris ac, consectetur bibendum lectus. Mauris non blandit odio. Maecenas et ultricies nulla. Nullam convallis mauris elementum, vehicula sem in, vehicula mi. Sed et urna non velit tincidunt fermentum non ac orci.</p>
+    </section>
+
+    <section id="sectionFour">
+        <p class="character-entities purple">Character Entities: &cent &pound &yen &amp</p>
```
Comment (mattbowytz) [inline index.html]: HTML entities need a semicolon at the end to be considered valid `&cent;` etc

File: index.html [hunk 5]
```diff
@@ -2,10 +2,60 @@
...
+    </section>
+
+    <fieldset class="">
```
Comment (mattbowytz) [inline index.html]: best not to leave empty attributes in HTML

File: index.html [hunk 6]
```diff
@@ -2,10 +2,60 @@
...
+      <label>First Name</label> <input type="text">
+      <label>Last Name</label> <input type="text">
+      <label>Age</label> <input type="number">
+    </fieldset>
+
+    <footer>
+      <a href="#">Link 4</a>
+      <a href="#">Link 5</a>
+      <a href="#">Link 6</a>
+      <p>Chris Lagunilla &copy 2016</p>
```
Comment (mattbowytz) [inline index.html]: same note here about HTML entities

**Comment by mattbowytz** [comment]
👍

**Comment by mattbowytz** [comment]
@chrislagunilla please close this pr whenever you get a chance

**Comment by matty-digital** [comment]
disregard the last comment, I was signed in under my other account

**Pull request closed without merging**